src/data.rs

```diff
@@ -206,7 +206,13 @@ pub struct SlotMetaTile {
     pub data: SlotMetaTileData,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct DataSourceDescription {
+    pub source_locator: Option<String>,
+}
```

Comment:
`DataSourceDescription` seemed like an OK name since "summary" and "info" are already taken, but lmk if there is something else you'd prefer. Also `Option` seemed like a more rust-y choice than e.g. letting an empty string be a signifier.

Reply:
The name seems fine to me and I agree that I prefer `Option` to assigning sentinel values (like `""` or `-1`).